README.md

```diff
@@ -218,5 +228,10 @@ Currently **officially** using Amundsen:
 1. [WeTransfer](https://wetransfer.com)
 1. [Workday](https://www.workday.com/en-us/homepage.html)
 
+## Contributors
+
+This project won't be possible without the help of a long list of [Contributors](https://github.com/amundsen-io/amundsen/blob/master/CONTRIBUTING.md), join them by following our [Contributing guide](https://github.com/amundsen-io/amundsen/blob/master/CONTRIBUTORS.md)!
```

Comment:
oh, should the first one be `CONTRIBUTORS.md` and the second one be `CONTRIBUTING.md` ?